src/generic.YAML-tmLanguage

```diff
@@ -0,0 +1,39 @@
+repository:
+    generic:
+        patterns:
+            -
+                begin: '(?=\<(?:[A-Z_]|\*|in|out))'
+                end: '(?<=\>)(?!\>)'
+                patterns:
+                    -
+                        match: '<'
+                        name: punctuation.bracket.angle.begin.kotlin
+                    -
+                        match: '\*'
+                        name: storage.modifier.generic.wildcard.kotlin
+                    -
+                        include: '#generic-parameter-list'
+                    -
+                        match: ','
+                        name: punctuation.seperator.kotlin
+                    -
+                        match: '>'
+                        name: punctuation.bracket.angle.end.kotlin
+
+        repository:
+            generic-parameter-list:
+                patterns:
+                    -
+                        include: '#annotations'
+                    -
+                        match: '\b(in|out)\b'
+                        name: storage.modifier.generic.variance.kotlin
+                    -
+                        include: '#built-in-types'
+                    -
+                        include: '#class-ident'
+                    -
+                        include: '#generic'
+                    -
+                        match: ':'

```

Comment:
`:` is referenced here https://github.com/nishtahir/language-kotlin/blob/e2b759590ff405dc86f2b8c39ff1dfa4848dae71/src/keywords.YAML-tmLanguage#L52-L54 as a declaration punctuation. Might be worth adding that for consistency. Storage modifier is typically used with keywords like `private` or `static`